### ISSUE TITLE
Display the number of quote posts under a post

### ISSUE THREAD

**Issue by simonetthomas** [bluesky-social/social-app#1249]
**Is your feature request related to a problem? Please describe.**

When viewing a post, there is currently no way of knowing if it has been quoted, and how many times.

**Describe the solution you'd like**

It would be nice if it displayed the number of times it has been quoted beside the number of reposts and likes, similarly to Twitter.
An additionnal feature would be to access these quote posts by tapping the number, like on Twitter, but displaying the number would already be nice.

**Describe alternatives you've considered**

I guess it could be added to the number of reposts, as was the case on Twitter years ago, but this can be misleading as reposts and quote posts are not exactly the same thing.

**Additional context**

**Comment by mschwendener** [comment]
> access these quote posts 

[1050](https://github.com/bluesky-social/social-app/issues/1050)

**Comment by pfrazee** [comment]
Yeah same as #1050, I'm categorizing this as "discussing" but I think we're on a good path with some other tools to reduce dunk quotes, so this may be likely

**Comment by AurelWu** [comment]
I'm also strongly in favour of this. While I see dunk-posts as an relevant issue, quote posts are very helpful to link discussions together and visibility of that is very useful. Unquoting sounds like a good approach, and maybe also have a preemptive option to disallow quote posts for either a specific post or as account wide setting?

**Comment by mschwendener** [comment]
This can be closed, no?
